great_expectations/data_context/data_context.py

```diff
@@ -742,11 +743,27 @@ def datasources(self) -> Dict[str, Union[LegacyDatasource, BaseDatasource]]:
 
     @property
     def checkpoint_store_name(self):
-        return self.project_config_with_variables_substituted.checkpoint_store_name
+        try:
+            return self.project_config_with_variables_substituted.checkpoint_store_name
+        except AttributeError as e:
+            if not default_checkpoints_exist(directory_path=self.root_directory):
```

Comment:
I don't think the `not` should be here, right? We want to re-raise if we don't find non-empty checkpoint dir

Reply:
Great catch.  This is the reason for the last minute errors.  Fixing now.

Reply:
Fixed.